model/src/main/java/io/neow3j/model/types/TransactionType.java

```diff
@@ -4,27 +4,29 @@
 
 public enum TransactionType {
 
-    MINER_TRANSACTION("MinerTransaction", 0x00),
-    ISSUE_TRANSACTION("IssueTransaction", 0x01),
-    CLAIM_TRANSACTION("ClaimTransaction", 0x02),
-    DATA_FILE("DataFile", 0x12),
-    ENROLLMENT_TRANSACTION("EnrollmentTransaction", 0x20),
-    REGISTER_TRANSACTION("RegisterTransaction", 0x40),
-    CONTRACT_TRANSACTION("ContractTransaction", 0x80),
-    RECORD_TRANSACTION("RecordTransaction", 0x81),
-    STATE_TRANSACTION("StateTransaction", 0x90),
-    STATE_UPDATE_TRANSACTION("StateUpdateTransaction", 0x90),
-    STATE_UPDATER_TRANSACTION("StateUpdaterTransaction", 0x91),
-    DESTROY_TRANSACTION("DestroyTransaction", 0x18),
-    PUBLISH_TRANSACTION("PublishTransaction", 0xd0),
-    INVOCATION_TRANSACTION("InvocationTransaction", 0xd1);
+    MINER_TRANSACTION("MinerTransaction", 0x00, 0),
+    ISSUE_TRANSACTION("IssueTransaction", 0x01, 0),
+    CLAIM_TRANSACTION("ClaimTransaction", 0x02, 0),
+    DATA_FILE("DataFile", 0x12, 0),
+    ENROLLMENT_TRANSACTION("EnrollmentTransaction", 0x20, 0),
+    REGISTER_TRANSACTION("RegisterTransaction", 0x40, 0),
+    CONTRACT_TRANSACTION("ContractTransaction", 0x80, 0),
+    RECORD_TRANSACTION("RecordTransaction", 0x81, 0),
+    STATE_TRANSACTION("StateTransaction", 0x90, 0),
+    STATE_UPDATE_TRANSACTION("StateUpdateTransaction", 0x90, 0),
+    STATE_UPDATER_TRANSACTION("StateUpdaterTransaction", 0x91, 0),
+    DESTROY_TRANSACTION("DestroyTransaction", 0x18, 0),
+    PUBLISH_TRANSACTION("PublishTransaction", 0xd0, 0),
+    INVOCATION_TRANSACTION("InvocationTransaction", 0xd1, 0);
 
     private String jsonValue;
     private byte byteValue;
+    private byte version;
 
-    TransactionType(String jsonValue, int v) {
+    TransactionType(String jsonValue, int v, int version) {
```

Comment:
The version was added here because, maybe, other transaction types could come up with other versions? If yes, then it's fine.